LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do we really want to include this file in versioning ?
I would rather have a dedicated tools subdirectory, and store all of them as a zip file